CMakeLists.txt

```diff
@@ -9,6 +9,10 @@ if(MSVC)
   set(CMAKE_GENERATOR_CC cl)
 endif()
 
+if(ARCH STREQUAL "aarch64" AND CMAKE_GENERATOR MATCHES "Visual Studio" AND NOT "${CMAKE_VS_PLATFORM_TOOLSET}" MATCHES "ClangCL")
```

Comment:
Is this issue aarch64 specific? Should this also check for arch "arm"?

Reply:
Yeah, I suspect Windows/ARM32 assembly would also need special handling, but don't know that we would even generate proper assembly for it. (I didn't consider Windows/ARM32 to be something had support for.)  We can revisit supporting that platform if anyone requests.